packages/cfstream/cfstream.1.3.2/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+homepage: "https://github.com/biocaml/cfstream"
+dev-repo: "git+https://github.com/biocaml/cfstream.git"
+bug-reports: "https://github.com/biocaml/cfstream/issues"
+license: "LGPL + linking exception"
+maintainer: "Ashish Agarwal <agarwal1975@gmail.com>"
+synopsis: "Stream operations in the style of Core's API"
+authors: [
+  "Philippe Veber <philippe.veber@gmail.com>"
+  "Ashish Agarwal <agarwal1975@gmail.com>"
+  "Drup <drupyog@zoho.com>"
+]
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "dune" {>= "1.0"}
+  "core_kernel" {>= "v0.11.0"}
+  "camlp-streams" {>= "5.0.1"}
```

Comment:
```suggestion
  "camlp-streams" {>= "5.0.1"}
  "conf-m4"
```
```

#=== ERROR while compiling cfstream.1.3.2 =====================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/biocaml/cfstream/archive/refs/tags/1.3.2.tar.gz)
# path                 ~/.opam/4.04/.opam-switch/build/cfstream.1.3.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p cfstream -j 127
# exit-code            1
# env-file             ~/.opam/log/cfstream-7-dfc905.env
# output-file          ~/.opam/log/cfstream-7-dfc905.out
### output ###
# File "/home/opam/.opam/4.04/lib/ppx_inline_test/ppx_inline_test.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "/home/opam/.opam/4.04/lib/ppx_expect/ppx_expect.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
# File "lib/dune", line 17, characters 41-43:
# 17 |  (action (with-stdout-to %{targets} (run m4 %{x})))
#                                               ^^
# Error: Program m4 not found in the tree or in PATH
#  (context: default)
```